app/src/highlighter/globals.d.ts

```diff
@@ -216,14 +216,19 @@ declare namespace CodeMirror {
    * id will be the id for the defined mode. Typically, you should use this second argument to defineMode as your module scope function
    * (modes should not leak anything into the global scope!), i.e. write your whole mode inside this function.
    */
+  // tslint:disable-next-line:no-unused-function
```

Comment:
TSLint lets you disable for over an area, so you can replace this one `/* tslint:disable:no-unused-function */` and insert `/* tslint:enable:no-unused-function */` where you want the rule back, then remove the other disable lines for the same effect.